month/months.js

```diff
@@ -1,70 +1,93 @@
-"use strict"
+import { isSameDate, setSelectedDate, selectedDate, WEEKDAYS } from "../js/dateManipulation.js"
+import { switchWeekView } from "../js/weekView.js"
+
 
 const CALENDAR_ROWS = 6
-const DAYS_IN_WEEK = 7
 
 const today = new Date(Date.now())
-let currentMonth = today.getMonth()
-let currentYear = today.getFullYear()
+let selectedDate_month = selectedDate
```

Comment:
something is wrong with variable naming. I'm not sure what it's supposed to be.
selectedMonth maybe?